### PR TITLE
feat: docs site

### PR DESCRIPTION
**Pull request by krishanu2911**
Created the Documentation site of the maple component library.

https://user-images.githubusercontent.com/75560307/153462387-6dbd1941-44c9-4ce2-ab01-15e8aec61415.mp4

### CODE REVIEW

File: CSS/typography.css [hunk 1]
```diff
@@ -69,11 +69,17 @@
...
-    height: 10rem;
+/* text - color */
+.header-text-color{
+    color: var(--headline-color);
+}
+.header-text-color:hover{
+    color: var(--paragraph-color);
+}
+.test-border{
+    /* width: 10rem; */
```
Comment (Sameony) [inline CSS/typography.css]: Remove unnecessary comments

File: CSS/typography.css [hunk 2]
```diff
@@ -69,11 +69,17 @@
 .border-rad-full{
     border-radius: 62.4375rem;
 }
-
-/* .test-border{
-    width: 10rem;
-    height: 10rem;
+/* text - color */
```
Comment (Sameony) [inline CSS/typography.css]: such comments are not really needed as the name of the class is descriptive enough

File: Doc/alert.html [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+                <div class="component-display-div maple-flex-xy-center">
+                    <div class="alert alert-success ">
+                        <i class="fas fa-check-square"></i>
+                        <h5 class="alert-message">This is the alert message you have notification</h5>
+                    </div>
+                    <div class="alert alert-fail">
+                        <i class="fas fa-bomb"></i>
+                        <h5 class="alert-message">This is the alert message you have notification</h5>
+                    </div>
+                    <div class="alert  alert-warn">
```
Comment (Sameony) [inline Doc/alert.html]: aha, found a 2 space difference between classes-
refactor the code properly-

File: Doc/alert.html [hunk 4]
```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../CSS/base.css">
+    <link rel="stylesheet" href="../Doc-css/docs.css">
+    <title>Document</title>
```
Comment (Sameony) [inline Doc/alert.html]: Do add relevant titles to the webpages

File: Doc/avatar.html [hunk 5]
```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../CSS/base.css">
+    <link rel="stylesheet" href="../Doc-css/docs.css">
+    <title>Document</title>
```
Comment (Sameony) [inline Doc/avatar.html]: Title change

File: Doc/card.html [hunk 6]
```diff
@@ -0,0 +1,312 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../CSS/base.css">
+    <link rel="stylesheet" href="../Doc-css/docs.css">
+    <title>Document</title>
```
Comment (Sameony) [inline Doc/card.html]: Correct this in every html file

File: Doc/card.html [hunk 7]
```diff
@@ -0,0 +1,312 @@
...
+                    <a href="./snackbar.html"><li class="list-pills">Snackbar</li></a>
+                    <a href="./grid.html"><li class="list-pills">Grid</li></a>
+                </ul>
+            </div>
+        </div>
+        <!-- main content area -->
+
+        <div class="main-display ">
+                <h1 class="margin-top-bottom">Card</h1>
+                <p class="simebold-font">This component is used to give the brif introduction about the product or and things gist within a container</p>
```
Comment (Sameony) [inline Doc/card.html]: spelling mistake: brief*

File: Doc/card.html [hunk 8]
```diff
@@ -0,0 +1,312 @@
...
+                <h1 class="margin-top-bottom">Card</h1>
+                <p class="simebold-font">This component is used to give the brif introduction about the product or and things gist within a container</p> 
+                <h2 class="margin-top-bottom">Badge Card</h2>
+                <div class="light-background component-display-div-row maple-flex-xy-center ">
+                    <div class="card card-relative">
+                        <div class="header">
+                            <div class="header-image-text-div">
+                                    <img class="header-image" src="../Assets/images/dore.jpg" alt="" srcset="">
+                                    <!-- <img class="header-image" src="https://picsum.photos/200" alt="" srcset=""> -->
+                            </div>
```
Comment (Sameony) [inline Doc/card.html:68]: unnecessary comment

File: Doc/card.html [hunk 9]
```diff
@@ -0,0 +1,312 @@
...
+                        </div>
+                    </div>
+                </div>
+                <h2 class="margin-top-bottom">Dismiss card</h2>
+                <div class="light-background component-display-div-row maple-flex-xy-center">
+                    <div class="card card-relative card-dismiss">
+                        <div class="header">
+                            <div class="header-image-text-div">
+                                    <img class="header-image" src="../Assets/images/dore.jpg" alt="" srcset="">
+                                    <!-- <img class="header-image" src="https://picsum.photos/200" alt="" srcset=""> -->
```
Comment (Sameony) [inline Doc/card.html]: Skim through the pages and remove unnecessary comments

File: Doc/card.html [hunk 10]
```diff
@@ -0,0 +1,312 @@
...
+                            
+                        </div>
+                        <div class="footer-card">
+                                <button class="btn ">Add to card</button>
+                                <button class="btn btn-secondary">Buy Now</button>
+                                <!-- <div class="icons">READ</div>
+                                <div class="icons">BOOKMARK</div>
+                                <i class="far fa-heart icons"></i>
+                                <i class="fas fa-external-link-alt icons"></i>
+                                <i class="fas fa-ellipsis-v icons"></i> -->
```
Comment (Sameony) [inline Doc/card.html]: this piece as well

File: Doc/image.html [hunk 11]
```diff
@@ -0,0 +1,127 @@
...
+                <div class="component-display-div-row maple-flex-xy-center">
+                    <div class="img">
+                        <img class="img-res" src="../Assets/images/sample-img.png" alt="" srcset="">
+                    </div>
+                </div>
+                
+                <h2 class="margin-top-bottom">Squared (rounded border) Image</h2>
+                <div class="component-display-div-row maple-flex-xy-center">
+                    <div class="img">
+                        <img class="img-res img-br-rounded" src="../Assets/images/sample-img.png" alt="" srcset="">
```
Comment (Sameony) [inline Doc/image.html]: provide alt text for images

File: Doc/list.html [hunk 12]
```diff
@@ -0,0 +1,133 @@
...
+                    <a href="./snackbar.html"><li class="list-pills">Snackbar</li></a>
+                    <a href="./grid.html"><li class="list-pills">Grid</li></a>
+                </ul>
+            </div>
+        </div>
+        <!-- main content area -->
+
+        <div class="main-display ">
+                <h1 class="margin-top-bottom">List</h1>
+                <p class="simebold-font">List is used to make an organized sturcture of showing data or notification and anything. so Maple has two varition of list.</p>
```
Comment (Sameony) [inline Doc/list.html]: spelling error: variation*
structure*

File: Doc/snackbar.html [hunk 13]
```diff
@@ -0,0 +1,124 @@
...
+                    <a href="./snackbar.html"><li class="list-pills"><h4>Snackbar</h4></li></a>
+                    <a href="./grid.html"><li class="list-pills">Grid</li></a>
+                </ul>
+            </div>
+        </div>
+        <!-- main content area -->
+
+        <div class="main-display ">
+                <h1 class="margin-top-bottom">Snackbar</h1>
+                <p class="simebold-font">This is cute component which don't dis turbs you gently informs you about the message.</p>
```
Comment (Sameony) [inline Doc/snackbar.html]: disturb*
grammar is actually wrong on so many levels-

File: index.html [hunk 14]
```diff
@@ -10,24 +10,25 @@
...
-                <div class="navbar-right">
-                    <a href="" class="navbar-icon txt-lg">Docs</a>
-                    <a href=""><i class="fab fa-github txt-xl navbar-icon"></i></a>  
-                </div>
-            </nav>
             <div class="main-content-area">
                 <div class="text-div">
                     <div class="para-div">
-                        <p class="paragraph ">Maple UI is a CSS Component library, provides ready-to-use frontend components that you can easily combine to build web interfaces.</p>
+                        <p class="paragraph ">Maple UI is Open Source Component Library which is light weight and responsive which creaters all the need for you site development. This is the one stop solution for your components related requirement.</p>
```
Comment (Sameony) [inline index.html:27]: caters to all the needs for your website development*

**Review by Sameony — COMMENTED** [review]
Suggested...juuuust a few changes....

File: Doc-css/docs.css [hunk 15]
```diff
@@ -0,0 +1,108 @@
...
+    border-radius: 0.5rem;
+}
+@media only screen and (max-width: 800px) {
+    .side-section{
+        display: none;
+    }
+    .main-display{
+        padding: 1rem ;
+    }
+    main{
```
Comment (nishasen) [inline Doc-css/docs.css:95]: is this main tag aur main class

File: Doc/alert.html (hunk 4 above)
Comment (coldpigli) [inline Doc/alert.html]: Change titles according to pages

**Review by coldpigli — COMMENTED** [review]
Added some comments

**Review by nishasen — COMMENTED** [review]
Everything looks good, just a minor change is needed. Please look into the comment

File: CSS/typography.css (hunk 1 above)
Comment (krishanu2911) [reply CSS/typography.css]: Done cleared it

File: CSS/typography.css (hunk 2 above)
Comment (krishanu2911) [reply CSS/typography.css]: ya cleared the comments

File: Doc/alert.html (hunk 3 above)
Comment (krishanu2911) [reply Doc/alert.html]: removed extra space 😅

File: Doc/alert.html (hunk 4 above)
Comment (krishanu2911) [reply Doc/alert.html]: Added title

File: Doc/avatar.html (hunk 5 above)
Comment (krishanu2911) [reply Doc/avatar.html]: Done will do for the all just forget to update thankyou for pointing out

File: Doc/card.html (hunk 7 above)
Comment (krishanu2911) [reply Doc/card.html]: corrected

File: Doc/card.html (hunk 8 above)
Comment (krishanu2911) [reply Doc/card.html:68]: done

File: Doc/card.html (hunk 9 above)
Comment (krishanu2911) [reply Doc/card.html]: Done removed it

File: Doc/snackbar.html (hunk 13 above)
Comment (krishanu2911) [reply Doc/snackbar.html]: ya sry for inconvenience will correct it

File: index.html (hunk 14 above)
Comment (krishanu2911) [reply index.html:27]: done

File: Doc-css/docs.css (hunk 15 above)
Comment (krishanu2911) [reply Doc-css/docs.css:95]: ya it's a main tag

File: Doc/alert.html (hunk 4 above)
Comment (krishanu2911) [reply Doc/alert.html]: Ya Thanks for pointing out corrected it

**Review by nishasen — APPROVED** [review]

**Review by coldpigli — APPROVED** [review]
Addressed the changes. good ahead and merge

**Review by Sameony — APPROVED** [review]

**Pull request merged**